src/main/java/racingcar/controller/RacingCarController.java

```diff
@@ -0,0 +1,25 @@
+package racingcar.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import racingcar.model.RacingResponse;
+import racingcar.model.RacingRequest;
+import racingcar.service.RacingCarService;
+
+@RestController
+public class RacingCarController {
+    private final RacingCarService racingCarService;
+
+    public RacingCarController(RacingCarService racingCarService) {
+        this.racingCarService = racingCarService;
+    }
+
+    @PostMapping("/plays")
+    public ResponseEntity<RacingResponse> racingGame(@RequestBody RacingRequest racingRequest) {
+        return ResponseEntity.ok(racingCarService.startRacing(racingRequest));
```

Comment:
메서드 체이닝을 이용해서 가독성을 높여보면 어떨까요?
```suggestion
        RacingResponse racingResponse = racingCarService.startRacing(racingRequest);
        return ResponseEntity.ok().body(racingResponse);
```